### PR TITLE
JBPM-8948 RepoServiceIntegrationTest failing due to different list of…

### PR DESCRIPTION
**Pull request by bxf12315**
… WIH present

### CODE REVIEW

File: repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java [hunk 1]
```diff
@@ -151,21 +153,24 @@ public void testAddServiceTask(){
 
     @Test
     public void testRemoveServiceTask() {
-        Optional<RepoData> beforeRemoveOne = repoService.getServices().stream().filter(s -> s.getName().equals(SERVICE_TASK_NAME_ONE)).findFirst();
+        assertTrue(getInitRepoDataList().size() > 2);
+        String service_task_name_one = getInitRepoDataList().get(0).getName();
```
Comment (jstastny-cz) [inline repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java]: Naming of the variables.

Comment (mareknovotny) [reply repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java]: What convention for naming is preferred? Could we fix it to not block the merge or do it later?

Comment (jstastny-cz) [reply repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java]: In Java? Camel case.
service_task_name_one -> serviceTaskNameOne ...

Comment (bxf12315) [reply repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java]: Done

**Pull request merged**